src/messaging/ExchangeMgr.cpp

```diff
@@ -72,13 +74,16 @@ CHIP_ERROR ExchangeManager::Init(SecureSessionMgr * sessionMgr)
 
     mNextExchangeId = GetRandU16();
 
+    memset((void *) mContextPool, 0, sizeof(mContextPool));
```

Comment:
I would feel more comfortable with a for loop that sets everything to nullptr.

The reason is that mContextPool here it is an array and in other places it is a plain pointer, so it is hard to determine if this code is correct or not.

The loop would also avoid a c style cast.

Reply:
Acked